stripe/src/main/java/com/stripe/android/view/StripeEditText.kt

```diff
@@ -30,7 +30,7 @@ open class StripeEditText @JvmOverloads constructor(
 
     private var afterTextChangedListener: AfterTextChangedListener? = null
     private var deleteEmptyListener: DeleteEmptyListener? = null
-    var cachedColorStateList: ColorStateList? = null
+    var cachedColorStateList: ColorStateList
```

Comment:
Can we leave this as nullable? I have some FUD about making it non-null.

Reply:
See comment below